src/guide/advanced/queries.md

```diff
@@ -66,6 +66,22 @@ Roles are an optional feature that should be present on all Iroha 2 deployments
 
   <WarningFatQuery />
 
+### FindAllRoleIds
+
+- **Returns**: `Vec<Roles>`
+
+- **Details**: It returns all roles registered as _global_ (as opposed to domain-scoped) in the blockchain, but instead of returning the values of Roles (which contain permission tokens), we only get the Ids of the roles.
+
+
+### FindRoleByRoleId
+
+- **Parameters**: `RoleId`
+
+- **Returns**: `Vec<Roles>`
+
+- **Details**: It returns the role that has the provided role ID. For example given the name of the role `admin` it will return all of the `admin`-level permission tokens.
```

Comment:
```suggestion
- **Details**: It returns the role that has the provided role ID. For example, given the name of the role `admin` it will return all of the `admin`-level permission tokens.
```